prac_07/project.py

```diff
@@ -0,0 +1,25 @@
+class Project:
+    """Represent a Project object."""
+
+    def __init__(self, name, start_date, priority, cost_estimate, completion_percentage):
+        """Initialise a Project instance."""
+        self.name = name
+        self.start_date = start_date
+        self.priority = priority
+        self.cost_estimate = cost_estimate
+        self.completion_percentage = completion_percentage
+
+    def __repr__(self):
+        """Return a string containing project information."""
+        return f"{self.name}, start: {self.start_date}, priority {self.priority}, estimate: ${self.cost_estimate}, " \
+               f"completion: {self.completion_percentage}%"
+
+    def __gt__(self, other):
```

Comment:
If you make the changes I suggest in project_management.py in regards to the date. You can use this method to compare priorities of projects. This will allow you to eaily sort projects by priority when displaying them.